ui/raidboss/data/07-dt/trial/necron-ex.ts

```diff
@@ -0,0 +1,631 @@
+        const offset = parseInt(bottomActorCount[0], 16) - 0x3B8;
+
+        data.cropCircleOrder = [...data.reapingSafeDirs, ...data.reapingSafeDirs].slice(
+          offset,
+          offset + 4,
+        );
+      },
+    },
+    {
+      id: 'NecronEx The Second/Fourth Season',
+      type: 'StartsUsing',
+      netRegex: { id: ['B06F', 'B070'], capture: true },
+      durationSeconds: 18,
+      infoText: (data, matches, output) => {
+        const [dir1, dir2, dir3, dir4] = data.cropCircleOrder;
+
+        if (
+          data.cropCircleOrder.length !== 4 || dir1 === undefined || dir2 === undefined ||
+          dir3 === undefined || dir4 === undefined
+        ) {
+          console.error(
+            `Crop Circle Collector: Invalid safe dir info`,
+            data.cropCircleOrder,
+          );
+          return;
+        }
+        const mech = matches.id === 'B06F' ? 'healerStacks' : 'partners';
+
+        return output.text!({
+          dir1: output[dir1]!(),
+          dir2: output[dir2]!(),
+          dir3: output[dir3]!(),
+          dir4: output[dir4]!(),
+          mech: output[mech]!(),
+        });
+      },
+      run: (data) => {
+        data.cropCircleActors = {};
+        data.cropCircleActors = {};
+        data.reapingSafeDirs = [];
+        data.cropCircleOrder = [];
+        data.cropCircleActors = {};
+      },
+      outputStrings: {
+        in: Outputs.in,
+        out: Outputs.out,
+        sides: Outputs.sides,
+        mid: Outputs.middle,
+        healerStacks: Outputs.healerGroups,
+        partners: Outputs.stackPartner,
+        text: {
+          en: '${dir1} => ${dir2} => ${dir3} => ${dir4} + ${mech}',
+        },
+      },
+    },
+    {
+      id: 'NecronEx Circle of Lives',
+      type: 'StartsUsing',
+      netRegex: { id: 'AE38', capture: true },
+      preRun: (data) => data.circleOfLivesCounter++,
+      delaySeconds: 0.2,
+      durationSeconds: 6.5,
+      infoText: (data, matches, output) => {
+        const pos = data.actorPositions[matches.sourceId];
+
+        if (pos === undefined) {
+          console.error(
+            `Circle of Lives: Missing actor data for ${matches.sourceId}`,
+            data.actorPositions,
+          );
+          return;
+        }
+
+        let safe: DirectionOutput8 | 'middle';
+
+        if (Math.abs(pos.x - 100) < 1)
+          safe = 'middle';
+        else
+          safe = Directions.xyTo8DirOutput(pos.x, pos.y, 100, 100);
+
+        // First 5 are part of the first set of mechs
+        if (data.circleOfLivesCounter <= 5) {
+          const handRow = data.loomingSpecterLocs[0];
+          const circleRow: LoomingSpecterDir = safe === 'middle'
+            ? 'middle'
+            : (pos.y < 99 ? 'north' : 'south');
+          // If this is the 2nd circle, and the hand is in line with the circle, warn to move quickly after the hit
+          if (data.circleOfLivesCounter === 2 && handRow === circleRow)
+            return output.dodge!({ dir: safe });
```

Comment:
I think this is supposed to be using the output string value, like this:
```suggestion
            return output.dodge!({ dir: output[safe]!() });
```

Won't be able to live test until Sunday but with that change it looks good in raidemulator.